src/nodule.js

```diff
@@ -44,9 +45,18 @@ export default class Nodule {
             loadFromObject(defaults),
             ...this.loaders,
         );
-        return loader(metadata).then((config) => {
-            bind('config', () => config);
-            return config;
+
+        const declassifiedLoader = loadEach(
+            loadFromObject(defaults),
+            ...this.declassifiedLoaders,
+        );
+
+        return declassifiedLoader(metadata).then((declassifiedConfig) => {
+            bind('declassifiedConfig', () => declassifiedConfig);
+            return loader(metadata).then((config) => {
```

Comment:
Future optimization: load every config only once

Reply:
I'll add this as a comment